index.html

```diff
@@ -0,0 +1,250 @@
+                    <li><a href="#" class="pro-two-a">bootstrap</a></li>
+                    <li><a href="#" class="pro-two-a">Ruby</a></li>
+                </ul>
+                <div class="view-project">
+                    <a href="#" id="view-project-id">See Project</a>
+                </div>
+            </div>
+            <div class="project project-three">
+                <h3>Website Protfolio</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li><a href="#" class="pro-three-a">html</a></li>
+                    <li><a href="#" class="pro-three-a">bootstrap</a></li>
+                    <li><a href="#" class="pro-three-a">Ruby</a></li>
+                </ul>
+                <div class="view-project">
+                    <a href="#" id="view-project-id">See Project</a>
+                </div>
+            </div>
+            <div class="project project-four">
+                <h3>Profesional Art <br> Printing Data More</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li><a href="#" class="pro-four-a">html</a></li>
+                    <li><a href="#" class="pro-four-a">bootstrap</a></li>
+                    <li><a href="#" class="pro-four-a">Ruby</a></li>
+                </ul>
+                <div class="view-project">
+                    <a href="#" id="view-project-id">See Project</a>
+                </div>
+                
+            </div>
+            <div class="project project-five">
+                <h3>Data Dashboard <br> Healthcare</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li><a href="#" class="pro-five-a">html</a></li>
+                    <li><a href="#" class="pro-five-a">bootstrap</a></li>
+                    <li><a href="#" class="pro-five-a">Ruby</a></li>
+                </ul>
+                <div class="view-project">
+                    <a href="#" id="view-project-id">See Project</a>
+                </div>
+            </div>
+            <div class="project project-six">
+                <h3>Website Protfolio</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li><a href="#" class="pro-six-a">html</a></li>
+                    <li><a href="#" class="pro-six-a">bootstrap</a></li>
+                    <li><a href="#" class="pro-six-a">Ruby</a></li>
+                </ul>
+                <div class="view-project">
+                    <a href="#" id="view-project-id">See Project</a>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] **[OPTIONAL]** Another aesthetic feature of the design for the desktop is the effects that should occur whenever users/visitors hover over any project card. In the image below, you will notice the `cursor` indicating the hovering which sets the `display` properties of other elements to none, removes the overlay in the background and displays the action button: 

![image](https://user-images.githubusercontent.com/99715640/214162431-5e9f70cb-1aad-4c92-9be7-63d7a73dd337.png)

- Project 1 is different because user hovers over it otherwise it should be like others. Implementing this for your design will further improve the aesthetic view of your page, exuding great impression on recruiters or viewers.  You gat this :muscle: